### PR TITLE
Add IAM

### PR DESCRIPTION
**Pull request by kdeldycke**
**https://github.com/kdeldycke/awesome-iam**

**Why is this list awesome?**

Backend development might be tedious and full of pitfalls. This awesome list focus on the whole Identity and Access Management  (IAM for shorts), and help you out with concepts, tutorial and tools about managing accounts, users, roles, permissions, authentication and authorization.

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

✅ I have read this PR template multiple times. Better yet: [I contributed to refining it](https://github.com/sindresorhus/awesome/commit/488c1051a3089e386ed223a5b45ab8ff8c53397c).

## Requirements for your pull request

- [x] **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.
- [x] **You have to review at least 2 other open pull requests.**
  Sort of. I'm involved in the community by other means:
    ✅ I already [contributed to this repository](https://github.com/sindresorhus/awesome/commits?author=kdeldycke).
    ✅ I [pushed code to `awesome-lint`](https://github.com/sindresorhus/awesome-lint/commits?author=kdeldycke) to automate even more the work of reviewing lists>
- [x] You have read and understood the instructions for creating a list.
- [x] This pull request has a title in the format `Add Name of List`.
- [x] Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
- [x] Your entry should be added at the bottom of the appropriate category.
- [x] The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- [x] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
  ✅ This repository [has been active since June 2019](https://github.com/kdeldycke/awesome-iam/commit/2982a50158ad86ab58d4788ae8f1b7808f99799e).
- [x] Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.
- [x] Run `awesome-lint` on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please report it.
- [x] The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- [x] **Includes a succinct description of the project/theme at the top of the readme.**
- [x] It's the result of hard work and the best I could possibly produce.
  ✅ I worked on this for 2 years, made 400+ commits, highly curated the list (we're far from a simple collection of links), and was involved into the community.
- [x] The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
- [x] The heading title of your list should be in title case format: `# Awesome Name of List`.
- [x] Non-generated Markdown file in a GitHub repo.
- [x] The repo should have `awesome-list` & `awesome` as GitHub topics. I encourage you to add more relevant topics.
- [x] Not a duplicate. Please search for existing submissions.
- [x] Only has awesome items. Awesome lists are curations of the best, not everything.
- [x] Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [x] Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- [x] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [x] Includes the [Awesome badge](awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- [x] Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- [x] Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- [x] Has [contribution guidelines](awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- [x] All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- [x] Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- [x] Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- [x] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [x] Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by kdeldycke** [comment]
🦄

**Comment by sindresorhus** [comment]
>  Includes a succinct description of the project/theme at the top of the readme.

File: readme.md [hunk 1]
```diff
@@ -313,6 +313,7 @@
 - [Vapor](https://github.com/Cellane/awesome-vapor#readme) - Server-side development in Swift.
 - [Dash](https://github.com/ucg8j/awesome-dash#readme) - Python web app framework.
 - [FastAPI](https://github.com/mjhea0/awesome-fastapi#readme) - Python web app framework.
+- [IAM](https://github.com/kdeldycke/awesome-iam#readme) - Identity and Access Management.
```
Comment (sindresorhus) [inline readme.md:316]: The description needs to elaborate more on the topic. And should not be title-case.

**Comment by sindresorhus** [comment]
The code of conduct is not valid.

**Comment by kdeldycke** [comment]
> Includes a succinct description of the project/theme at the top of the readme.

Done in https://github.com/kdeldycke/awesome-iam/commit/03789f607669fbd4a481f8260cfbbe536484537e

**Comment by kdeldycke** [comment]
> The code of conduct is not valid.

Do you have any suggestion on what could make a good code of conduct? Should I reuse the one in the canonical `awesome` repository?

File: readme.md (hunk 1 above)
Comment (kdeldycke) [reply readme.md:316]: I have several proposals:
* Expand the acronym in the title: `Identity and Access Management - <new description>.`
* Keep it that way: `IAM - <new description>.`

Which one you prefer?

Comment (kdeldycke) [reply readme.md]: As for the new description, I pushed it in: https://github.com/sindresorhus/awesome/pull/1916/commits/43ac52c294bd44bbd229f362dec855f39717a2fa

**Comment by kdeldycke** [comment]
> The code of conduct is not valid.

Code of conduct has just been updated in https://github.com/kdeldycke/awesome-iam/commit/383bacb563288183022997377ed1e96dd3ac8ab0 to the same Contributor Covenant v1.4 used in Awesome project.

File: readme.md (hunk 1 above)
Comment (kdeldycke) [reply readme.md]: I choose to keep the `IAM` acronym as I found a couple of non-expended domain-specific acronyms in the awesome list (`IoT`, `CTF`, `GDPR`, `LIDAR`, `OKR`, ...), so I guess it is a reasonable choice.

**Comment by kdeldycke** [comment]
All review comments on this PR have been addressed and this PR is ready to be merged! :)

**Comment by sindresorhus** [comment]
Linkify `IAM` in `IAM stands for Identity and Access Management. But is more than handling user accounts:` to https://en.wikipedia.org/wiki/Identity_management

**Comment by kdeldycke** [comment]
> Linkify `IAM` in `IAM stands for Identity and Access Management. But is more than handling user accounts:` to https://en.wikipedia.org/wiki/Identity_management

Done upstream in https://github.com/kdeldycke/awesome-iam/commit/2094628d245406cd3df221262825a15d15c40130 .

Thanks @sindresorhus for the suggestion! :)

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1368570386430881798

**Comment by kdeldycke** [comment]
Thanks a lot @sindresorhus! You rock! :)

**Pull request merged**